src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/action/CloseActionConfig.kt

```diff
@@ -0,0 +1,57 @@
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.model.action
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.action.Action
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.action.CloseAction
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.ManagedIndexMetaData
+import org.elasticsearch.client.Client
+import org.elasticsearch.cluster.service.ClusterService
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.ToXContentObject
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParserUtils
+import java.io.IOException
+
+data class CloseActionConfig(
+    val timeout: ActionTimeout?,
+    val retry: ActionRetry?,
+    val index: Int
+) : ToXContentObject, ActionConfig(ActionType.CLOSE, timeout, retry, index) {
+
+    override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
+        builder.startObject().startObject(ActionType.CLOSE.type)
+        super.toXContent(builder, params)
+        return builder.endObject().endObject()
+    }
+
+    override fun isFragment(): Boolean = super<ToXContentObject>.isFragment()
```

Comment:
Is there a reason you preferred calling the ```super``` implementation rather than explicitly specifying ```false```? I assumed we defined this method for readability but since ```ToXContentObject```'s implementation returns ```false```, defining this and calling ```super``` would be redundant.

Reply:
Resolving this since Yan made a similar comment.